### PR TITLE
Translated seo chapter to Japanese.

### PR DESCRIPTION
**Pull request by MSakamaki**
translation of seo chapter and fixed of #542

### CODE REVIEW

File: src/content/ja/2019/seo.md [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+
+#### 見出し
+
+また、ページに含まれるコンテンツのコンテキストが適切な方法で構造化されて提供されているかを調べました。
+見出し（`H1`、 `H2`、 `H3`、など）を使ってページを整え構造化すれば、コンテンツは読みやすく、解析しやすいようになります。
+そんな見出しの重要性にもかかわらず、ページの10.67%には見出しタグがまったくありませんでした。
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig2.png">
+    <img src="/static/images/2019/10_SEO/fig2.png" alt="Figure 2. Distribution of the number of headings per page." aria-labelledby="fig2-caption" aria-describedby="fig2-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=676369575&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig2.png" alt="図2.ページ毎の見出し数分布。" aria-labelledby="fig2-caption" aria-describedby="fig2-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=676369575&amp;format=interactive">
```

File: src/content/ja/2019/seo.md [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+しかし、検索エンジンは現在どのようにコンテンツを見つけるのでしょうか？、その回答を得るために、単語数と見出しの２つのカスタムメトリクスを作りました。
+
+#### 単語の数
+
+私達は、少なくとも３つの単語グループを探し合計でいくつ見つかったかを数えるようにして、ページのコンテンツを評価しました。
+デスクトップページには単語グループを持たないものが2.73%見つかりました。これはWebサイトが何を指しているのかを検索エンジンが理解するのに役立つ本文コンテンツが無いことを示しています。
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig1.png">
+    <img src="/static/images/2019/10_SEO/fig1.png" alt="Figure 1. Distribution of the number of words per page." aria-labelledby="fig1-caption" aria-describedby="fig1-caption" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=190546113&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig1.png" alt="図1.ページ毎の単語数分布。" aria-labelledby="fig1-caption" aria-describedby="fig1-caption" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=190546113&amp;format=interactive">
```

File: src/content/ja/2019/seo.md [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+  <figcaption id="fig2-caption" >図2.ページ毎の見出し数分布。</figcaption>
+</figure>
+
+１ページあたりの見出し要素の中央値は10となっています。
+見出しにはモバイルページで30単語、デスクトップページで32単語が含まれています。
+これは、見出しを活用できているWebサイトが、ページが読みやすく、説明的で、ページの構造とコンテキストを検索エンジンボットに明確に概説することに多大な労力を費やしていることを意味します。
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig3.png">
+    <img src="/static/images/2019/10_SEO/fig3.png" alt="Figure 3. Distribution of H1 length per page." aria-labelledby="fig3-caption" aria-describedby="fig3-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1380411857&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig3.png" alt="図3.ページ毎のH1の長さの分布。" aria-labelledby="fig3-caption" aria-describedby="fig3-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1380411857&amp;format=interactive">
```

File: src/content/ja/2019/seo.md [hunk 4]
```diff
@@ -0,0 +1,522 @@
...
+  <div class="big-number">97%</div>
+  <figcaption>図4. <code>&lt;title></code>タグを含むモバイルページの割合。</figcaption>
+</figure>
+
+ページのタイトルはページの目的をユーザーや検索エンジンに伝える重要な手段です。
+`<title>`タグはSERPSの見出にも、ページにアクセスする時のブラウザーのタブのタイトルとしても使われるので、モバイルページの97.1%にドキュメントタイトルが存在することは驚くことではないでしょう。
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig5.png">
+    <img src="/static/images/2019/10_SEO/fig5.png" alt="Figure 5. Distribution of title length per page." aria-labelledby="fig5-caption" aria-describedby="fig5-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1015017335&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig5.png" alt="図5.ページごとのタイトルの長さの分布。" aria-labelledby="fig5-caption" aria-describedby="fig5-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1015017335&amp;format=interactive">
```

File: src/content/ja/2019/seo.md [hunk 5]
```diff
@@ -0,0 +1,522 @@
...
+これは、一部のSEOとコンテンツの記者が、検索エンジンによって割り当てられたSERPsのホームページを記述するために割り当てられた領域を最大限利用できていないことを示します。
+
+#### メタディスクリプション
+
+`<title>`タグと比べると、メタディスクリプションが検出されたページは少なくなっており、モバイル用ホームページの64.02%にだけメタディスクリプションが設定されています。
+Googleが検索者のクエリに応じてSERP内のメタディスクリプションの記述を頻繁に書き換えることを考慮すると、おそらくWebサイトの所有者はメタディスクリプションを含めることを重要視しないでしょう。
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig6.png">
+    <img src="/static/images/2019/10_SEO/fig6.png" alt="Figure 6. Distribution of meta description length per page." aria-labelledby="fig6-caption" aria-describedby="fig6-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1750266149&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig6.png" alt="図6. ページ毎のメタ記述の長さ分布。" aria-labelledby="fig6-caption" aria-describedby="fig6-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1750266149&amp;format=interactive">
```

File: src/content/ja/2019/seo.md [hunk 6]
```diff
@@ -0,0 +1,522 @@
...
+内部や外部リンクを持たないごく一部のページは、ターゲットページへ通じるリンクという大きな価値を取りこぼしています。
+
+デスクトップ用のページに含まれる内部と外部リンクの数は、モバイル用のページよりも全ての場合で多くなっています。これは殆どの場合、モバイルのデザインはビューポートが小さく空間が限られているために、リンクが含まれるテキストはデスクトップに比べて少なくなっているためです。
+
+モバイル用のページで内部リンクが少ない場合、Webサイトで[問題が発生する可能性](https://moz.com/blog/internal-linking-mobile-first-crawl-paths)が有るため注意が必要です。
+新しいWebサイトでGoogleの規定である[モバイルファーストインデックス](https://www.deepcrawl.com/knowledge/white-papers/mobile-first-index-guide/)が適用されると、そのページがデスクトップ用ではリンクされているがモバイル用からリンクが無い時、検索エンジンはそのページを見つけてランク付けするのがとても難しくなってしまいます。
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig7.png">
+    <img src="/static/images/2019/10_SEO/fig7.png" alt="Figure 7. Distribution of internal links per page." aria-labelledby="fig7-caption" aria-describedby="fig7-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=534496673&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig7.png" alt="図7.ページ毎の内部リンク分布。" aria-labelledby="fig7-caption" aria-describedby="fig7-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=534496673&amp;format=interactive">
```

File: src/content/ja/2019/seo.md [hunk 7]
```diff
@@ -0,0 +1,522 @@
...
+  </a>
+  <div id="fig7-description" class="visually-hidden">ページ毎の内部リンク数の分布。デスクトップの内部リンクは10、25、50、75、90パーセンタイルごとに、7、29、70、142、261となっています。モバイル分布はかなり低く、90パーセンタイルでリンク数は30、中央値で10となっています。</div>
+  <figcaption id="fig7-caption" >図7.ページ毎の内部リンク分布。</figcaption>
+</figure>
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig8.png">
+    <img src="/static/images/2019/10_SEO/fig8.png" alt="Figure 8. Distribution of external links per page." aria-labelledby="fig8-caption" aria-describedby="fig8-description" width="600" data-idth="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1997009875&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig8.png" alt="図8.ページ毎の外部リンク数の分布。" aria-labelledby="fig8-caption" aria-describedby="fig8-description" width="600" data-idth="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1997009875&amp;format=interactive">
```

File: src/content/ja/2019/seo.md [hunk 8]
```diff
@@ -0,0 +1,522 @@
...
+  </a>
+  <div id="fig8-description" class="visually-hidden">ページ毎の外部リンク数の分布。デスクトップの外部リンクは10、25、50、75、90パーセンタイルごとに、1、4、10、22、51となっています。モバイルの分布はかなり低く、90パーセンタイルでリンク数は11、中央値で2となっています。</div>
+  <figcaption id="fig8-caption" >図8.ページ毎の外部リンク数の分布。</figcaption>
+</figure>
+
+デスクトップ用ページの内部リンク(同一サイト)数は中央値で70となっていますが、モバイル用ページの内部リンク数の中央値は60になっています。外部リンク数のページ毎中央値も同じような傾向となっており、デスクトップ用ページの外部リンク数は10で、モバイル用ページは8になっています。
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig9.png">
+    <img src="/static/images/2019/10_SEO/fig9.png" alt="Figure 9. Distribution of anchor links per page." aria-labelledby="fig9-caption" aria-describedby="fig9-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1852200766&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig9.png" alt="図9.ページ毎のアンカーリンク数の分布。" aria-labelledby="fig9-caption" aria-describedby="fig9-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=1852200766&amp;format=interactive">
```

File: src/content/ja/2019/seo.md [hunk 9]
```diff
@@ -0,0 +1,522 @@
...
+Webサイトが高速であることは、優れたユーザー体験のためにも重要です。
+サイトの読み込みに数秒待たされるユーザは、すぐ離脱してSERPsから別の似たような内容の素早く読み込まれるページを探す傾向があります。
+
+Web全体の読み込み速度の分析に使った指標は [Chrome UX Report](./methodology#chrome-ux-report)（CrUX）を基にしています。このレポートは、実際のChromeユーザーからデータを収集します。
+このデータで驚くべき点は、48%のWebサイトが**遅い**とラベル付されていることです。
+FCPの25%が3秒より遅い場合、_もしくは_ FIDの5%が300ミリ秒より遅い場合にWebサイトは低速とラベル付されます。
+
+<figure>
+  <a href="/static/images/2019/10_SEO/fig10.png">
+    <img src="/static/images/2019/10_SEO/fig10.png" alt="Figure 10. Distribution of the performance of user experiences by device type." aria-labelledby="fig10-caption" aria-describedby="fig10-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=2083126642&amp;format=interactive">
```
Comment (bazzadp) [inline src/content/ja/2019/seo.md]:
```suggestion
    <img src="/static/images/2019/10_SEO/fig10.png" alt="図10.デバイスタイプごとのユーザー体験パフォーマンスの分布。" aria-labelledby="fig10-caption" aria-describedby="fig10-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSrPaauIA_G4AySC09FX4fK5DsJ8DWhJGUQE0obrBe9HGSA8geyq3KwFi531jg9Ll9auE3x_UEwnF8g/pubchart?oid=2083126642&amp;format=interactive">
```

**Comment by MSakamaki** [comment]
@bazzadp 
Thank you for review! 
I fixed alt tags.

**Pull request merged**